matisse/src/main/res/values-ru/strings.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's very nice of you to add Russian and Ukrainian strings. But would you please add these localizations for the sample app?